### PR TITLE
Email for reporting tool

### PR DESCRIPTION
**Pull request by ryuk156**
Fixes  https://github.com/CircuitVerse/CircuitVerse/issues/1465

#### Describe the changes you have made in this PR -
I have added a email text field in the reporting tool using html and fetch the value of email text field using jquery and concatenate it with message variable 

### Screenshots of the changes (If any) -

Reporting tool
image

Slack:
image

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

**Comment by ryuk156** [comment]
@nitin10s  @satu0king @tachyons  review and merge please

File: app/views/simulator/edit.html.erb [hunk 1]
```diff
@@ -319,6 +319,10 @@
     <hr>
      <div class="container my-2 text-center" id="result">
     </div>
+    <label id="email-label" style="font-weight: lighter" ><b>Email:</b></label>
```
Comment (nitin10s) [inline app/views/simulator/edit.html.erb:322]: give a for="emailtext" attribute to this label as well.

**Comment by ryuk156** [comment]
@nitin10s  @tachyons @satu0king  review and merge please

**Comment by ryuk156** [comment]
> image
> 
> This is what I see when I tried it ...

I made changes can you check it again please it will work now

**Pull request merged**